host/xtest/regression_1000.c

```diff
@@ -3270,7 +3270,69 @@ static void xtest_tee_test_1039(ADBG_Case_t *c)
 	Do_ADBG_EndSubCase(c, "Load TA with identity subkey");
 
 }
-
-
 ADBG_CASE_DEFINE(regression, 1039, xtest_tee_test_1039,
 		 "Test subkey verification");
+
+struct test_1040_thread_arg {
+	TEEC_Result res;
+	pthread_t thr;
+};
+
+static void *test_1040_thread(void *arg)
+{
+	TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
+	struct test_1040_thread_arg *a = arg;
+	TEEC_Result res = TEEC_SUCCESS;
+	uint32_t err_orig = 0;
+	TEEC_Session session = { };
+	size_t n = 0;
+
+	while (n < 100) {
+		res = xtest_teec_open_session(&session, &sims_test_ta_uuid,
+					      NULL, &err_orig);
+		if (res) {
+			if (res == TEEC_ERROR_TARGET_DEAD)
+				continue;
+			a->res = res;
+			return NULL;
+		}
+
+		memset(&op, 0, sizeof(op));
+		op.paramTypes = TEEC_PARAM_TYPES(TEEC_MEMREF_TEMP_INPUT,
+						 TEEC_NONE, TEEC_NONE,
+						 TEEC_NONE);
+		res = TEEC_InvokeCommand(&session, TA_SIMS_CMD_PANIC, &op,
+					 &err_orig);
+		TEEC_CloseSession(&session);
+		if (res != TEEC_ERROR_TARGET_DEAD) {
+			a->res = res;
+			return NULL;
+		}
+		n++;
+	}
+	return NULL;
```

Comment:
explicitly set `a->res = TEE_SUCCESS` to clarifies the test at line 3331?

Reply:
OK